Doc/whatsnew/3.14.rst

```diff
@@ -70,6 +70,78 @@ Summary -- Release highlights
 New Features
 ============
 
+.. _whatsnew-314-pep649:
+
+PEP 649: Deferred Evaluation of Annotations
+-------------------------------------------
+
+The :term:`annotations <annotation>` on functions, classes, and modules are no
+longer evaluated eagerly. Instead, annotations are stored in special-purpose
+:term:`annotate functions <annotate function>` and evaluated only when
+necessary. This is specified in :pep:`649` and :pep:`749`.
+
+The new :mod:`annotationlib` module provides tools for inspecting deferred
+annotations. Annotations may be evaluated in the :attr:`~annotationlib.Format.VALUE`
+format (which evaluates annotations to runtime values, similar to the behavior in
+earlier Python versions), the :attr:`~annotationlib.Format.FORWARDREF` format
+(which replaces undefined names with special markers), and the
+:attr:`~annotationlib.Format.SOURCE` format (which returns annotations as strings).
+
+This example shows how these formats behave::
+
+   from annotationlib import get_annotations, Format
+
+   def func(arg: Undefined):
+       pass
+
+   print(get_annotations(func, format=Format.VALUE))  # raises NameError
+   print(get_annotations(func, format=Format.FORWARDREF))  # {'arg': ForwardRef('Undefined')}
+   print(get_annotations(func, format=Format.SOURCE))  # {'arg': 'Undefined'}
+
+Implications if you write annotations in your code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
Implications for writers of annotations
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I'm not sure about this, it sort of sounds like we're talking about code generation tools. I was struggling with how to communicate "people who just put annotations on their function and run a type checker".

Reply:
Hmm, yeah. Maybe "Implications for annotated code"...?

Reply:
That's a good one, let's go with that.
```suggestion
Implications for annotated code
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```